bower.json

```diff
@@ -8,7 +8,7 @@
     "angular": "~1.4.2",
     "angular-animate": "~1.4.2",
     "angular-aria": "~1.4.2",
-    "angular-material": "~0.11.4",
+    "angular-material": "~0.12.0-rc1",
```

Comment:
There are errors on the console on the replica set list page. Remove flex attr from one of the buttons.